packages/dialtone-vue2/components/hovercard/hovercard.vue

```diff
@@ -0,0 +1,201 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<template>
+  <dt-popover
+    :id="id"
+    :placement="placement"
+    :content-class="contentClass"
+    :fallback-placements="fallbackPlacements"
+    :padding="padding"
+    :transition="transition ? 'fade' : null"
+    :offset="offset"
+    :modal="false"
+    initial-focus-element="none"
+    :header-class="headerClass"
+    :footer-class="footerClass"
+    :append-to="appendTo"
+    :hovercard="true"
+    :timer="timer"
+    data-qa="dt-hovercard"
+    @opened="(e) => ($emit('opened', e))"
+  >
+    <template #anchor="{ attrs }">
+      <slot
+        name="anchor"
+        v-bind="attrs"
+      />
+    </template>
+    <template #content>
+      <slot name="content" />
+    </template>
+    <template #headerContent>
+      <slot name="headerContent" />
+    </template>
+
+    <template #footerContent>
+      <slot name="footerContent" />
+    </template>
+  </dt-popover>
+</template>
+
+<script>
+import { reactive } from 'vue';
+import { POPOVER_APPEND_TO_VALUES, POPOVER_PADDING_CLASSES, DtPopover } from '@/components/popover/index.js';
+import { getUniqueString } from '@/common/utils';
+import useTimer from './timer';
+
+export default {
+  name: 'DtHovercard',
+
+  components: {
+    DtPopover,
+  },
+
+  props: {
+  /**
+   * Fade transition when the content display is toggled.
+   * @type boolean
+   * @values true, false
+   */
+    transition: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+   * If the popover does not fit in the direction described by "placement",
+   * it will attempt to change its direction to the "fallbackPlacements".
+   * <a
+   *   class="d-link"
+   *   href="https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements"
+   *   target="_blank"
+   * >
+   *   Popper.js docs
+   * </a>
+   * */
+    fallbackPlacements: {
+      type: Array,
+      default: () => {
+        return ['auto'];
+      },
+    },
+
+    /**
+   * The direction the popover displays relative to the anchor.
+   * <a
+   *   class="d-link"
+   *   href="https://atomiks.github.io/tippyjs/v6/all-props/#placement"
+   *   target="_blank"
+   * >
+   *   Tippy.js docs
+   * </a>
```

Comment:
This is a styled doc for our storybook, I will suggest to move all this links to the `.mdx` file and have here the short description for the component.

Reply:
Do you have an example of a component where this is done? I see the docs are filled automatically from these comments

Reply:
I mean, we are affecting the component prop section for storybook purpose.

Thats the point of adding stuff like this:
 ```
* <a
   *   class="d-link"
   *   href="https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements"
   *   target="_blank"
   * >
   *   Popper.js docs
   * </a>
   * */
```
<img width="883" alt="Screenshot 2024-01-11 at 15 18 35" src="https://github.com/dialpad/dialtone/assets/89984179/c07bdd07-3ccd-470c-a0ec-0b573f3e836a">

What I suggest is to do something like this:

<img width="1221" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/fc959c78-92c5-4b29-af1b-7e5cac4e25ec">


Then we avoid to create noise in component prop description.

It is just a suggestion it is not necessary to be done on this way, but just heads up we are interfering component props doc for storybook purposes

Reply:
Fixed in latest changes

Reply:
looks great